quinn-proto/src/connection.rs

```diff
@@ -3251,6 +3281,11 @@ const MAX_ACK_BLOCKS: usize = 64;
 pub enum EndpointEvent {
     ClientReady,
     Incoming,
+    Closed {
+        init_cid: Option<ConnectionId>,
```

Comment:
`Connection` never reads this value, so I think it would now make more sense for it to be stored in the `Endpoint`'s connection table directly rather than kept inside the `Connection` and passed back in an event.

Reply:
Done.